authors/apps/core/exceptions.py

```diff
@@ -23,6 +23,7 @@ def core_exception_handler(exc, context):
 
     return response
 
+
```

Comment:
There is not change here. So it is unnecessary

Reply:
pep8 requires 2 extra lines, it found one so we added the other one